torch_xla/csrc/torch_util.cpp

```diff
@@ -32,4 +32,10 @@ at::Tensor UnwrapNumber(const at::Tensor& tensor, at::ScalarType dtype) {
                                                            : tensor;
 }
 
+bool IsNonTrivialDilation(at::IntArrayRef dilation) {
```

Comment:
I would just move this in the new autograd files.

Reply:
It's also used in `aten_xla_type.cpp` so it'll have to be shared...

Reply:
Yeah, what I meant, you can use it from there as well.
Just instead of being in torch_util, it's in the new place.

Reply:
Oh ic! Fixing...